src/background.js

```diff
@@ -1642,9 +1648,9 @@ function initializeGhosteryModules() {
 				conf.enable_ad_block = !adblocker.isDisabled;
 				conf.enable_anti_tracking = !antitracking.isDisabled;
 				conf.enable_human_web = !humanweb.isDisabled;
-				conf.enable_offers = !offers.isDisabled && !IS_ANDROID;
+				conf.enable_offers = !offers.isDisabled && !IS_ANDROID && BROWSER_INFO.name !== 'ghostery_desktop';
```

Comment:
Is there a potential race here? How early is this block evaluated in the startup? The browser name will not be correct until after `chrome.runtime.getBrowserInfo` has resolved.

Reply:
This evaluates pretty far down the init chain and is nested inside `cliqz.start()`. The line directly below uses a similar check for android and has been working ok. But I admit I don't love this implementation. Open to suggestions.

Reply:
We could attach a `Globals.browserInfoReady` Promise in `Globals` when we call `buildBrowserInfo`, which we can then just `await` at these points that need to be sure that the browser check has been done.